test/circuits/binmulfast1.circom

```diff
@@ -1,5 +1,5 @@
 pragma circom 2.0.0;
 
-include "../../circuits/binmulfast.circom";
```

Comment:
Shouldn't we also change the filename for tests from binmulfast1 to chunkedmul1?